v2/tools/generator/internal/armconversion/convert_to_arm_function_builder.go

```diff
@@ -7,6 +7,7 @@ package armconversion
 
 import (
 	"fmt"
+	"github.com/pkg/errors"
```

Comment:
minor: ordering, should go with `dave/dst` below

Reply:
Moved.